mlrun/model_monitoring/controller.py

```diff
@@ -69,16 +69,25 @@ def _get_last_analyzed(self) -> Optional[int]:
                 key=self._application,
             )
         except HttpResponseError as err:
-            logger.warn(
+            logger.info(
                 "Failed to get the last analyzed time for this endpoint and application, "
-                "as this is probably the first time this application is running. "
-                "Using the first request time instead.",
+                "as this is probably the first time this application is running. ",
+                "Using the latest between first request time or last update time minus one day instead.",
                 endpoint=self._endpoint,
                 application=self._application,
                 first_request=self._first_request,
+                last_update=self._stop,
                 error=err,
             )
-            return self._first_request
+
+            # TODO : Change the timedelta according to the policy.
+            first_period_in_seconds = max(
+                3600, self._step
```

Comment:
This is 1hr, change to:
```suggestion
                86400, self._step
```
Or even better:
```suggestion
                datetime.timedelta(days=1).total_seconds(), self._step
```

Reply:
@jond01 you review it to fast.. thanks anyway :)